scss/tech.scss

```diff
@@ -1,5 +1,8 @@
 .tech-container {
   padding: 10px 40px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
```

Comment:
margin in each `tech` should be enough. you can reduce the 1280px width if we want less items per row